app/javascript/packs/articlePage.jsx

```diff
@@ -124,8 +127,12 @@ getCsrfToken().then(async () => {
       root,
     );
   } catch (e) {
-    document.getElementById('comment-subscription').innerHTML =
-      '<p className="color-accent-danger">Unable to load Comment Subscription component.<br />Try refreshing the page.</p>';
+    const commentElement = document.getElementById('comment-subscription');
```

Comment:
question: we assigned (successfully) the root element earlier. This getElementById is either redundant (we have `root` and it points to the same thing) or error prone (what this change addresses) or wrong (if we're finding the comment subscription from a different article, it seems like we should _not_ be writing an error message to the comment subscription in the fresh page).

Would 

```javascript
  root.innerHTML = "<p..."
```

make as much sense, with the caveat that we don't get the error if it's not there?